spec/feature_flagger/notifier_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+
+module FeatureFlagger
+  RSpec.describe Notifier do
+    let(:feature_key)           { 'account:email_marketing:whitelabel' }
+    let(:legacy_feature_key)   { 'account' }
+    let(:resource_id)   { 'resource_id' }
+    let(:resource_name) { 'account' }
+
+    describe '#send' do
+      context 'When having a callback configured' do
+        let(:notifier_callback) { spy(lambda { |event| }, :is_a? => Proc)}
+        let(:notifier) { Notifier.new(notifier_callback)}
+        let(:generic_event) {
+          {
+            type: FeatureFlagger::Notifier::RELEASE,
+            model: resource_name,
+            feature: feature_key,
+            id: resource_id
+          }
+        }
+        context 'When call the lambda function' do
+          before { notifier.send(FeatureFlagger::Notifier::RELEASE, feature_key, resource_id) }
+
+          it { expect(notifier_callback).to have_received(:call) }
+        end
+
+        context 'When trigger the expected event' do
```

Comment:
What are we testing here? Why do we need to test every `feature_action` we have today?

Reply:
For each action, the notifier callback will receive a specific constant value, so I think is good to evaluate that each action is using the expected constant.